### ISSUE TITLE
NFT Collection description should be a single line input in create flow

### ISSUE THREAD

**Issue by wraeth-eth** [jbx-protocol/juice-interface#2779]
## Summary

The collection description field is currently a single-line text field. This does not allow for adequate description of collections and should be changed to a text area to allow for more detailed descriptions.

## Steps to reproduce

1. Navigate to NFT -> Add NFTs
2. Observe that the collection description field is a single-line text field

## What is the expected _correct_ behavior?

The collection description field should be a text area to allow for more detailed descriptions.

## Relevant logs and/or screenshots

![image](https://user-images.githubusercontent.com/104132113/212572576-79c91b31-135f-474d-a189-8542fcc7b121.png)

## Discord Link

https://discord.com/channels/775859454780244028/866040669712678942/1063556961372733531

**Comment by johnnyd-eth** [comment]
@wraeth-eth I remember that I did initally have a text-area here, but noticed that OpenSea was not-allowing and removing new-lines. So to not mislead users, I just made this a single-line field.

**Comment by wraeth-eth** [comment]
Looks like when you have new lines it does not get set properly at opensea. Confirmed this with @johnnyd-eth as well

**Comment by wraeth-eth** [comment]
Going to unassign myself from this one for now.

Update:
It appears that when new lines are included in the description, it doesn't call the function correctly. I haven't had a chance to fully wrap my head around how this works so can't confirm how to fix.

We have two options I think:
1) Work out why it isn't working with opensea wrt multiple lines
2) Change create to be single line to match the behavior here

**Comment by johnnyd-eth** [comment]
> Going to unassign myself from this one for now.
> 
> Update: It appears that when new lines are included in the description, it doesn't call the function correctly. I haven't had a chance to fully wrap my head around how this works so can't confirm how to fix.
> 
> We have two options I think:
> 
> 1. Work out why it isn't working with opensea wrt multiple lines
> 2. Change create to be single line to match the behavior here

I think option 2 for now. Happy to take this one @NakedFool ? LMK if you have any questions.

**Comment by wraeth-eth** [comment]
> > Going to unassign myself from this one for now.
> > Update: It appears that when new lines are included in the description, it doesn't call the function correctly. I haven't had a chance to fully wrap my head around how this works so can't confirm how to fix.
> > We have two options I think:
> > 
> > 1. Work out why it isn't working with opensea wrt multiple lines
> > 2. Change create to be single line to match the behavior here
> 
> I think option 2 for now. Happy to take this one @NakedFool ? LMK if you have any questions.
> 
> **TL;DR:**
> 
> * Change the collection description text-area field to a single-line text-input (where new-lines are not possible).
> * Doing this because OpenSea does not allow newlines in this description, and in fact, disregards the whole description if any new lines are present.

From googling around, it seems like it is possible, just not sure what the cost of finding out how to do it is worth it rn

**Comment by NakedFool** [comment]
> > Going to unassign myself from this one for now.
> > Update: It appears that when new lines are included in the description, it doesn't call the function correctly. I haven't had a chance to fully wrap my head around how this works so can't confirm how to fix.
> > We have two options I think:
> > 
> > 1. Work out why it isn't working with opensea wrt multiple lines
> > 2. Change create to be single line to match the behavior here
> 
> I think option 2 for now. Happy to take this one @NakedFool ? LMK if you have any questions.
> 
> **TL;DR:**
> 
> * Change the collection description text-area field to a single-line text-input (where new-lines are not possible).
> * Doing this because OpenSea does not allow newlines in this description, and in fact, disregards the whole description if any new lines are present.

Sure! 

Is the final decision to revert back to a single-line text input or investigate a little and see if we can apply the correct description with textarea?

**Comment by wraeth-eth** [comment]
> 

Yeah close it out with a single line text area change! The investigation can be done at a later date - low priority
